hexa/pipelines/schema/mutations.py

```diff
@@ -1,30 +1,16 @@
 import base64
-import pathlib
-
-from ariadne import (
-    EnumType,
-    MutationType,
-    ObjectType,
-    QueryType,
-    UnionType,
-    convert_kwargs_to_snake_case,
-    load_schema_from_path,
-)
+
+from ariadne import MutationType, convert_kwargs_to_snake_case
 from django.conf import settings
 from django.core.exceptions import PermissionDenied
 from django.db import IntegrityError
 from django.http import HttpRequest
-from django.urls import reverse
-from sentry_sdk import capture_exception
+from google.api_core.exceptions import NotFound
```

Comment:
I think you forgot to update this import (import NotFound exception from  hexa.files.api).